routes/users.rb

```diff
@@ -105,7 +114,15 @@
 
   user.update(is_member: true) || halt(500, ResponseFormat.error('Error updating user.'))
 
-  # TODO: initiate some sort of crowd script that adds them to the AD or w/e (if possible)
+  # Create an async task to add the user to the ACM Active Directory
+  ActiveDirectoryJob.perform_async(user.netid)
+
+  EmailNotificationJob.perform_async(
```

Comment:
What do you think about putting this in the `ActiveDirectoryJob` method so that the email only gets sent after it is successfully added to AD?